include/zephyr/bluetooth/services/ias.h

```diff
@@ -60,6 +68,43 @@ int bt_ias_local_alert_stop(void);
 #define BT_IAS_CB_DEFINE(_name)                                                                    \
 	static const STRUCT_SECTION_ITERABLE(bt_ias_cb, _CONCAT(bt_ias_cb_, _name))
 
+struct bt_ias_client_cb {
+	/** @brief Callback function for bt_ias_discover.
+	 *
+	 *  This callback is called when discovery procedure is complete.
+	 *
+	 *  @param conn Bluetooth connection object.
+	 *  @param err 0 on success, ATT error or negative errno otherwise
+	 */
+	void (*discover)(struct bt_conn *conn, int err);
+};
+
+/** @brief Set alert level
+ *
+ *  @param conn Bluetooth connection object
+ *  @param alert_lvl Alert level value to be written
+ *
+ *  @return Zero in case of success and error code in case of error.
+ *
+ */
+int bt_ias_alert_write(struct bt_conn *conn, enum bt_ias_alert_lvl alert_lvl);
```

Comment:
Given that the server has 3 callbacks instead of reporting the alert level, should the client follow the same convention? 3 functions instead of supplying the alert level?

Reply:
Or vice versa, reporting the alert level via a single callback on the server side?

Reply:
Done (added separate functions for different alert levels)